randovania/data/json_data/prime1.txt

```diff
@@ -196,207 +199,428 @@ Transport to Magmoor Caverns West
+      Scan Visor
+
+> Elevator - Transport to Phendrana Drifts North; Heals? False
+  * Teleporter to Magmoor Caverns - Transport to Phendrana Drifts North
+  > Dock to Shoreline Entrance
+      Trivial
 
 ----------------
 Shoreline Entrance
 Asset id: 3289414871
 > Dock to Transport to Magmoor Caverns West; Heals? False
   * Normal Door to Transport to Magmoor Caverns West/Dock to Shoreline Entrance
+  > Dock to Phendrana Shorelines
+      Missile ≥ 3 or Charge Beam
 
 > Dock to Phendrana Shorelines; Heals? False
   * Normal Door to Phendrana Shorelines/Dock to Shoreline Entrance
+  > Dock to Transport to Magmoor Caverns West
+      Missile ≥ 3 or Charge Beam
 
 ----------------
 Phendrana Shorelines
 Asset id: 4146616697
 > Dock to Shoreline Entrance; Heals? False
   * Normal Door to Shoreline Entrance/Dock to Phendrana Shorelines
+  > Dock to Temple Entryway
+      Space Jump Boots
+  > Dock to Save Station B
+      Trivial
+  > Dock to Plaza Walkway
+      Space Jump Boots
+  > Dock to Ice Ruins Access
+      Missile and Scan Visor
 
 > Dock to Temple Entryway; Heals? False
   * Normal Door to Temple Entryway/Dock to Phendrana Shorelines
+  > Dock to Shoreline Entrance
+      Trivial
 
 > Dock to Save Station B; Heals? False
   * Normal Door to Save Station B/Dock to Phendrana Shorelines
+  > Dock to Shoreline Entrance
+      Trivial
 
 > Dock to Ruins Entryway; Heals? False
   * Normal Door to Ruins Entryway/Dock to Phendrana Shorelines
+  > Dock to Shoreline Entrance
+      Trivial
+  > Dock to Plaza Walkway
+      Trivial
 
 > Dock to Plaza Walkway; Heals? False
   * Normal Door to Plaza Walkway/Dock to Phendrana Shorelines
+  > Dock to Shoreline Entrance
+      Trivial
+  > Dock to Ruins Entryway
+      Trivial
 
 > Dock to Ice Ruins Access; Heals? False
   * Normal Door to Ice Ruins Access/Dock to Phendrana Shorelines
+  > Dock to Shoreline Entrance
+      Trivial
+  > Pickup (Missile Behind Ice)
+      Plasma Beam
+  > Pickup (Spider Track Missile)
+      Scan Visor and Morph Ball and Spider Ball and Shoot Super Missile
+
+> Pickup (Missile Behind Ice); Heals? False
+  * Pickup 35; Major Location? False
+  > Dock to Ice Ruins Access
+      Trivial
+
+> Pickup (Spider Track Missile); Heals? False
+  * Pickup 36; Major Location? False
+  > Dock to Ice Ruins Access
+      Trivial
 
 ----------------
 Temple Entryway
 Asset id: 2245653401
 > Dock to Chozo Ice Temple; Heals? False
   * Normal Door to Chozo Ice Temple/Dock to Temple Entryway
+  > Dock to Phendrana Shorelines
+      Any of the following:
+          Plasma Beam or Missile
+          All of the following:
+              Charge Beam
+              Power Beam or Wave Beam
 
 > Dock to Phendrana Shorelines; Heals? False
   * Normal Door to Phendrana Shorelines/Dock to Temple Entryway
+  > Dock to Chozo Ice Temple
+      Any of the following:
+          Plasma Beam or Missile
+          All of the following:
+              Charge Beam
+              Power Beam or Wave Beam
```

Comment:
Does charged Ice have issues here, hence requiring a beam for charge?

Reply:
Yes, charged ice beam didn't work there. It does work on the other ice barriers in shorelines entryway, strangely enough.